src/App.js

```diff
@@ -1,25 +1,29 @@
-import logo from './logo.svg';
 import './App.css';
+import Counter from "./components/Counter";
+import SearchForm from "./components/SearchForm";
+import GenreSelect from "./components/GenreSelect";
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+export default function App() {
+    function handleSearch(searchInput) {
+        console.log("Input from the search bar: ", searchInput);
+    }
+
+    const genreNames = ['All', 'Documentary', 'Comedy', 'Horror', 'Crime'];
+
+    const selectedGenre = 'Documentary';
```

Comment:
Move constant to separate file, please. If this const only for this component we can hold it near the component, if we reuse const in different files, create shared => constants and put there. 
src
 components
  Input folder
    Input.js
    Input.css
    constants.js(for this component)
  shared folder
    constants